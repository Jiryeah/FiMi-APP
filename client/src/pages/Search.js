import React from 'react'
import SearchComp from '../components/SearchComp'

const Search = () => {
  return (
    <div>
      <SearchComp/>
    </div>
  )
}

export default Search