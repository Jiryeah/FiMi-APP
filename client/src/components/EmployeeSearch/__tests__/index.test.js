import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Search from '..';

afterEach(cleanup);

describe('Search Component', () => {
  // baseline Search test
  it('renders', () => {
    render(<Search />);
  });

  it('matches snapshot DOM node structure', () => {
    // render Search snapshot
    const { asFragment } = render(<Search />);
    expect(asFragment()).toMatchSnapshot();
  });
});
