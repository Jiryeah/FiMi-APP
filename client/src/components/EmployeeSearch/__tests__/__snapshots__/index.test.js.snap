// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    Search
  </div>
</DocumentFragment>
`;
